ServerlessLibraryAPI/ClientApp/src/components/Contribute/ContributionsList.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { Label } from "office-ui-fabric-react";
+import ItemList from "../../components/ItemList/ItemList";
+
+class ContributionsList extends Component {
+  filteredSamples() {
+    let { samples, user } = this.props;
+    let { userName } = user || "__nouser__";
+    let filter = new RegExp(userName, "i");
+    samples = samples.filter(
+      el => el.repository.replace("https://github.com/", "").match(filter) // this match should be against author
+    );
+
+    samples = samples.sort(function(a, b) {
```

Comment:
there is already a sort method. You can move these code to helpers and reuse at both place

Reply:
We don't need the sort as per the current specs. Will remove